README.md

```diff
@@ -1,126 +1,14 @@
-# template-microservice
-This serves as a template for the microservices
+# Flashcard Service
 
-## Package structure
+The Flashcard Service within the platform serves as an indispensable tool for facilitating and reinforcing repetitive learning, ensuring that students effectively retain the knowledge acquired throughout their course journey. 
+These digital flashcards emulate the concept of traditional physical flashcards, but with an added layer of interactivity.
 
-This package structure is based on multiple sources of best practices in Spring Boot, using roughly the "Package by layer" approach.
-- *root*
-  - *config*
-  - *controller*
-  - *dapr*
-  - *dto*
-  - *exception*
-  - *persistence*
-    - *dao*
-    - *repository*
-    - *mapper*
-  - *service*
-  - *util* (optional, if needed)
-  - *validation*
+Each flashcard embodies a two-sided structure, resembling the classic format. On one side, it presents educational content in various forms, such as text, images, or even videos, along with a thought-provoking question pertaining to the material that students have previously engaged with. 
+Lecturers manually create these questions to tailor the learning experience.
 
-Detailed description of the packages:
+Flipping to the other side of the virtual flashcard reveals the answer to the posed question. Here, students can gauge their understanding and knowledge retention by assessing whether they knew the answer beforehand or not. 
```

Comment:
This and the next section are just frontend details. That has nothing todo with this service.
Maybe describe what functionality the API provides?

Reply:
As I said like I followed our old wiki and that's why it becomes wrong, but I will change all services